gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -252,6 +253,34 @@ const NxToastPage = () =>
         </NxStatefulAccordion>
       </NxTile.Content>
     </NxTile>
+
+    <NxTile>
+      <NxTile.Header>
+        <NxTile.HeaderTitle>
+          <NxH2>NxDrawer With NxToast Example</NxH2>
+        </NxTile.HeaderTitle>
+      </NxTile.Header>
+      <NxTile.Content>
+        <NxP>
+          An example of <NxCode>NxDrawer</NxCode> with <NxCode>NxToast</NxCode>.
+          The toast is always positioned above NxDrawer, no matter the order in which they are opened.
+        </NxP>
+        <NxP>
+          <NxTextLink href="#/NxDrawerWithNxToastExample">
+            Click here to navigate to the live example.
+          </NxTextLink>
+        </NxP>
+      </NxTile.Content>
+      <NxTile.Content className= "nx-tile-content--accordion-container">
+        <NxStatefulAccordion>
+          <NxAccordion.Header>
+            <NxAccordion.Title>Example Code</NxAccordion.Title>
+          </NxAccordion.Header>
+          <CodeExample content={NxDrawerWithNxToastExampleCode} />
+          <GalleryTileFooter clipboardContent={NxDrawerWithNxToastExampleCode}/>
```

Comment:
Because the same NxDrawerWithNxToastExample is being used for Toast and Drawer, the back button brings you back to the NxDrawer page, no matter if you are looking at the home page from NxToast or NxDrawer

Reply:
> Because the same NxDrawerWithNxToastExample is being used for Toast and Drawer, the back button brings you back to the NxDrawer page, no matter if you are looking at the home page from NxToast or NxDrawer

I made a new example for the toast page: 7266925a146d039a3b8a1bfdaa6411ddf6021aff